cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[cypher-deprecations-additions-removals-compatibility]]
+= Deprecations, additions and compatibility
+
+* <<cypher-deprecations-additions-removals, Removals, deprecations, additions and extensions>>
+ ** <<cypher-deprecations-additions-removals-3.0, Version 3.0>>
+ ** <<cypher-deprecations-additions-removals-3.1, Version 3.1>>
+ ** <<cypher-deprecations-additions-removals-3.2, Version 3.2>>
+ ** <<cypher-deprecations-additions-removals-3.3, Version 3.3>>
+* <<cypher-compatibility, Compatibility>>
+ ** <<cypher-start-compatibility, Accessing entities by id via START>>
+* <<cypher-versions, Supported language versions>>
+
+[abstract]
```

Comment:
I noted that other docs in this folder have the abstract before the index and not after like this. Not important to me, simply pointing it out.

Reply:
Also the rendered version of the abstract looks block-quoted, which seems wrong to me.

Reply:
👍